docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -127,8 +127,8 @@
         },
         "grafanaDependency": {
           "type": "string",
-          "description": "Required Grafana version for this plugin, e.g. `>=7.0.0` to denote plugin requires Grafana 7.0.0 or later",
-          "pattern": "^(>=|<|>|<=)?([0-9]+)(\\.[0-9]+)?(\\.[0-9])?$"
+          "description": "Required Grafana version for this plugin. Validated using https://github.com/npm/node-semver.",
+          "pattern": "^(<=|>=|<|>|=|~|\\^)?([0-9]+)(\\.[0-9x\\*]+)(\\.[0-9x\\*])?(\\s(<=|>=|<|=>)?([0-9]+)(\\.[0-9x]+)(\\.[0-9x]))?$"
```

Comment:
This regular expression does not accept ranges like: `1.2.7 || >=1.2.9 <2.0.0` but I guess it is highly improbable to have gaps between the supported versions.

Reply:
Maybe we can expand the expression if it turns out to be too restrictive?

Reply:
It depends if we want to support such cases, otherwise it's fine.